GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -12,6 +12,7 @@ public partial class AppearanceSettingsPage : SettingsPageWithHeader
     {
         private readonly TranslationString _noDictFile = new TranslationString("None");
         private readonly TranslationString _noDictFilesFound = new TranslationString("No dictionary files found in: {0}");
+        private readonly TranslationString _noImageServiceTooltip = new TranslationString("A default image, if an email address has no matching Gravatar image.\r\nSee http://en.gravatar.com/site/implement/images/ for more details.");
```

Comment:
Not just gravatar, GitHub (may be) checked first

Reply:
The default image comes from Gravatar though, if we can't get an image from GH or GR.